thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -192,6 +192,13 @@ export default Component.extend({
    */
   metricGranularity: reads('anomaly.attributes.metricGranularity.firstObject'),
 
+  /**
+   * Anomaly detect rule
+   * @type {string}
+   */
+  detectRule: reads('anomaly.attributes.detectorComponentName'),
```

Comment:
What if it only shows the rule name?  For example, in the screenshot it shows "detector_rule_1:MIGRATED_ALGORITHM", but instead show just "detector_rule_1".  This is how we display the rule in Preview and Alert Overview.

Reply:
`detectRule: computed('anomaly.attributes.detectorComponentName', function () {
    return (get(this, 'anomaly').attributes.detectorComponentName || 'N/A:').split(':')[0];
  }),`

Reply:
This will grab the name only or, if it's not present, return 'N/A'

Reply:
Still doesn't work with split function. @harleyjj

Reply:
Got it fixed finally.